clang/test/Sema/address_spaces.c

```diff
@@ -1,3 +1,4 @@
+// XFAIL: *
```

Comment:
Do you think we can handle these tests without XFAIL? I mean, they support the common behavior of `address_space`, that is required for mainstream compatibility.

Reply:
Currently we have created a copy of these testes under AIE target (`clang/test/CodeGen/aie/address_space/`) and supported most of them. 
Based on discussion on LLVM forum & Internally (Documented internally on JIRA). Plan is to extend the support of `isAddressSpaceSupersetOf ` clang address_space to support requirement of AIE in parallel.

Reply:
Hi @krishnamtibrewala, understood, thank you for the explanation.